library/Icingadb/Widget/PerfdataTable.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Module\Icingadb\Util\Perfdata;
+use Icinga\Module\Icingadb\Util\PerfdataSet;
+use ipl\Html\Html;
+use ipl\Html\HtmlElement;
+use ipl\Html\HtmlString;
+use ipl\Html\Table;
+
+class PerfdataTable extends Table
+{
+    private $containsSparkline = false;
```

Comment:
Why is this private? protected should suffice, just like for the other attributes.

And please add a docblock, also to all other attributes you're introducing.